firebase/index.js

```diff
@@ -1,27 +1,24 @@
-const admin = require("firebase-admin");
+const jwt = require("jsonwebtoken");
 const config = require("../config");
 
 const firebase = { auth: {} };
 
-// credential: admin.credential.cert(credentials.get("firebase"))
-const defaultApp =
-  config.get("env") === "test_remote"
-    ? {
-        auth: () => {
-          console.log("temp stub");
-        },
-      }
-    : admin.initializeApp({
-        credential: admin.credential.cert({
-          project_id: config.get("firebase").project_id,
-          private_key: config.get("firebase").private_key.replace(/\\n/g, "\n"),
-          client_email: config.get("firebase").client_email,
-        }),
-      });
+// Function to generate JWT token
+const generateToken = (payload) => {
+  return jwt.sign(payload, config.get("jwt_secret"), { expiresIn: "1h" });
+};
 
-const auth = defaultApp.auth();
+// Function to verify JWT token
+const verifyToken = (token) => {
+  try {
+    return jwt.verify(token, config.get("jwt_secret"));
+  } catch (err) {
+    console.error("Token verification failed:", err);
+    return null;
+  }
+};
 
-firebase.auth = auth;
-firebase.admin = admin;
+firebase.generateToken = generateToken;
+firebase.verifyToken = verifyToken;
 
-module.exports = firebase;
+module.exports = firebase;
```

Comment:
this shouldn't be firebase either.

Reply:
yes i have changed that please check now